src/openapi-mcp-server/client/http-client.ts

```diff
@@ -3,6 +3,7 @@ import OpenAPIClientAxios from 'openapi-client-axios'
 import type { AxiosInstance } from 'axios'
 import FormData from 'form-data'
 import fs from 'fs'
+import { Headers } from './polyfill-headers'
```

Comment:
@mquan unfortunately we still need to import here from the polyfill to register it.

It will only provide the polyfill if the Headers class is not available globally already.